code/modules/surgery/robotics.dm

```diff
@@ -6,6 +6,9 @@
 /datum/surgery_step/robotics/
 	can_infect = 0
 	can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		if (isipc(target) && isipc(user) && user == target)
```

Comment:
`if (user == target && isipc(user))` just collapses this better.

Reply:
Fixed.